distribution/ddf-common/src/main/resources/security/configurations.policy

```diff
@@ -56,6 +56,11 @@ grant codeBase "file:/catalog-core-directorymonitor/org.apache.camel.camel-core/
      // permission java.io.FilePermission "/test/some/directory${/}-", "read";
 // }
 
+    // Example:
```

Comment:
It looks like master and the release branch were different here. This should be changed to: 
```suggestion
grant codeBase "file:/org.apache.tika.core/catalog-core-urlresourcereader" {
    //# Add permission to read the directory
    // permission java.io.FilePermission "${/}test${/}some${/}directory", "read";
    
    //# Add permission to recursively read the files and sub directories
    // permission java.io.FilePermission "${/}test${/}some${/}directory${/}-", "read";
}
```

Reply:
Applied your suggestion